addons/mail/static/src/rtc/rtc_service.js

```diff
@@ -435,6 +449,14 @@ export class Rtc {
         this.soundEffectsService.play("mute");
     }
 
+    async raiseHand() {
+        await this.setIsRaisingHand(true);
+    }
+
+    async lowerHand() {
+        await this.setIsRaisingHand(false);
+    }
```

Comment:
Not need to split in 2 functions.
Can just be a single function `raiseHand(true/false)`.
This solves issue of making a weirdly named function `setIsRaisingHand`.